executor/executor.go

```diff
@@ -1385,12 +1385,20 @@ func ResetContextOfStmt(ctx sessionctx.Context, s ast.StmtNode) (err error) {
 		sc.MemTracker.SetActionOnExceed(action)
 	}
 
+	// execute missed stmtID uses empty sql
+	sc.OriginalSQL = s.Text()
```

Comment:
Why do we move it up to here?

Reply:
Because when it's an `explainStmt`, `s` will be reset in line 1402.

Reply:
But it will also affects the execute stmt, is it right to change it here?

Reply:
But ExecStmt is not checked in this function.

Reply:
For `ast.ExecuteStmt`, before this PR, original sql would be the prepared stmt like "select * from ...", but after this PR, original sql would be `execute xx`. Am I right?

Reply:
I got you.